config/config.php

```diff
@@ -33,8 +35,10 @@
 	$mac_url = $url . 'Mac/Installer/';
 } else {
 	$url = 'https://github.com/nextcloud/desktop/releases/download/v' . $ver . '/';
+	$legacyurl = 'https://github.com/nextcloud/desktop/releases/download/v' . $legacy_ver . '/';
```

Comment:
Iirc versions before 3.1.x cant follow redirects and therefor need to be served from download.nextcloud.com

Reply:
Correct. This won't work for those.

Reply:
Thanks both, I will fix that